openerp/addons/base/ir/ir_translation.py

```diff
@@ -713,3 +713,27 @@ def load_module_terms(self, cr, modules, langs, context=None):
                     _logger.info('module %s: loading extra translation file (%s) for language %s', module_name, lang_code, lang)
                     tools.trans_load(cr, trans_extra_file, lang, verbose=False, module_name=module_name, context=context)
         return True
+
+
+    @api.model
+    def get_technical_translations(self, model):
+        fields = self.env['ir.model.fields'].search([('model', '=', model)])
+        trans = self.search([
+            '|',
+                '&',('type', '=', 'model'),
+                    '&',('res_id', 'in', fields.ids),
+                        '|',
+                            ('name', '=', 'ir.model.fields,field_description'),
+                            ('name', '=', 'ir.model.fields,help'),
+                '&',('type', '=', 'selection'),
+                    ('name', 'ilike', model),
```

Comment:
The `name` of translations of type "selection" is defined as `"%s,%s" % (model_name, field_name)`.
Therefore the condition on `name` should be something like

    `('name', 'like', model + ",")`

Otherwise translations of model `res.partner` will include translations of models `res.partner.category`, `res.partner.title`, `res.partner.bank`, etc.